docs/CHANGELOG.asciidoc

```diff
@@ -51,6 +52,222 @@ Review important information about the {kib} 8.x releases.
+* Adds permissions for "input-only" package ({kibana-pull}166234[#166234]).
+* Adds selecting the consumer based on the authorized consumers when a user is creating an ES Query threshold rule ({kibana-pull}166032[#166032]).
+* Migrate Ace based `EuiCodeEditor` to Monaco based code editor ({kibana-pull}165951[#165951]).
+* Mobile UI crash widget added ({kibana-pull}163527[#163527]).
+Cases::
+* Show a warning message to inform user that navigating after the 10Kth case is not possible ({kibana-pull}164323[#164323]).
+Dashboard::
+* Focus on a single panel while disabling all other panels ({kibana-pull}165417[#165417]).
+* Adds filter details to panel settings ({kibana-pull}162913[#162913]).
+Discover::
+* Redesign for the grid, panels and sidebar ({kibana-pull}165866[#165866]).
+* Set data table row height to auto-fit by default ({kibana-pull}164218[#164218]).
+* Allow fetching more documents on Discover page ({kibana-pull}163784[#163784]).
+* Fixes unified search for long field inputs ({kibana-pull}166024[#166024]).
+Elastic Security::
+For the Elastic Security 8.11.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Enterprise Search::
+For the Elastic Enterprise Search 8.11.0 release information, refer to {enterprise-search-ref}/changelog.html[_Elastic Enterprise Search Documentation Release notes_].
+Fleet::
+* Adds sidebar navigation showing headings extracted from the readme ({kibana-pull}167216[#167216]).
+Lens & Visualizations::
+* Other bucket defaults to false for top values greater than equal 1000 in *Lens* ({kibana-pull}167141[#167141]).
+* Adds support for decimals in percentiles in *Lens* ({kibana-pull}165703[#165703]).
+* Support date fields in options list ({kibana-pull}164362[#164362]).
+Machine Learning::
+* Update ELSER version for Elastic Assistant ({kibana-pull}167522[#167522]).
+* Retain `created_by` setting when exporting anomaly detection jobs ({kibana-pull}167319[#167319]).
+* Improved wording of awaiting ML nodes messages ({kibana-pull}167306[#167306]).
+* Adding `created_by` job property for the advanced wizard ({kibana-pull}167021[#167021]).
+* Trained model testing: only show indices with supported fields ({kibana-pull}166490[#166490]).
+* Alerts as data integration for Anomaly Detection rule type ({kibana-pull}166349[#166349]).
+* Data Frame Analytics Trained models: add ability to reindex after pipeline creation success ({kibana-pull}166312[#166312]).
+* Improvements to UX of adding ML embeddables to a dashboard ({kibana-pull}165714[#165714]).
+* AIOps: Support text fields in log rate analysis ({kibana-pull}165124[#165124]).
+* Data Frame Analytics creation wizard: adds ability to add custom URLs to jobs ({kibana-pull}164520[#164520]).
+Management::
+* Adds Create a data view button to index or saved search selector in ML pages and Transforms management ({kibana-pull}166668[#166668]).
```

Comment:
This PR applies to both the Management section (transforms) and ML. Can it be added to the Machine Learning section too?